ios/Sources/MediaFeature/MediaListFeature.swift

```diff
@@ -17,22 +17,28 @@ public struct MediaListState: Equatable {
     var blogs: [FeedContent]
     var videos: [FeedContent]
     var podcasts: [FeedContent]
-    var next: Next?
+    var isSearchResultVisible: Bool
+    var isSearchTextEditing: Bool
+    var isMoreActiveType: MediaType?
```

Comment:
I think `is**` is for condition variable name.
So how about rename this with `moreActiveType` ?